src/main/kotlin/net/ccbluex/liquidbounce/utils/aiming/PointTracker.kt

```diff
@@ -211,6 +216,7 @@ class PointTracker(
             .withMinY(lowest)
             .contract(shrinkBox.toDouble(), 0.0, shrinkBox.toDouble())
             .contract(speedShrinkFactor, abs(player.velocity.y), speedShrinkFactor)
+            .expand(boxExpand.toDouble(), 0.0, boxExpand.toDouble())
```

Comment:
Won't this make the rotations miss the target?

Reply:
not if you use straight box point

Reply:
... then expand is completely useless anyway